compliance/runners.py

```diff
@@ -180,7 +181,16 @@ def get_fetchers(self):
                         self.load_errors.add(load_err)
                 except AttributeError:
                     pass
-        return fetchers
+        include = {f'{f.__module__}.{f.__name__}' for f in fetchers}
```

Comment:
The amount of fetchers might be big enough here so calculating this even we don't using `include/exclude` might be a waste of CPU cycles. Perhaps I would add:

```
if not self.opts.include and not self.opts.exclude:
   return fetchers
...
```